recipes/common/silverblue-modules.yml

```diff
@@ -6,6 +6,7 @@ modules:
         - adw-gtk3-theme
         - gnome-epub-thumbnailer
         - gnome-tweaks
+        - sushi
```

Comment:
I agree with the removal of the two unverified flatpaks, but why is sushi needed? it seems like loupe serves this need

Reply:
It's not vital, but it's a feature of nautilus, so I figured for a vanilla Gnome experience it should be included. I've now removed it from the PR, so now this just fixes the unavailable flatpaks being requested.